src/internal/packager2/pull.go

```diff
@@ -31,6 +33,11 @@ import (
 
 // Pull fetches the Zarf package from the given sources.
 func Pull(ctx context.Context, src, dir, shasum string, filter filters.ComponentFilterStrategy, publicKeyPath string, skipSignatureValidation bool) error {
+	l := logger.From(ctx)
+	start := time.Now()
+	defer func() {
+		l.Debug("done Pull", "src", src, "dir", dir, "duration", time.Since(start))
```

Comment:
I like the intent here as I think (correct me) it meets some of the requests of late as exposing more information for consumers on what occurred and the results. 

is `done Pull` as eloquent of a message as we require? Concise and straight to the point.

Reply:
Ideally this would point to the function itself, e.g. `done packager2.Pull ...` but packager2 is a moving target so I just named it Pull for now. --log-format="dev" will also give us line numbers, IIRC.

Reply:
Added the pkg prefix in latest